docs/troubleshooting/troubleshoot_deployment.md

```diff
@@ -8,6 +8,18 @@ hide_title: true
 
 This document provides 'notes' on frequently encountered deployment issues. The issues reported are organized by relevant subsection.
 
+## Terraform Init
+
+The `TF_LOG` environment variable can be set to help debug Terraform-specific issues. See [Terraform debugging](https://www.terraform.io/docs/internals/debugging.html).
```

Comment:
I would specify that setting `TF_LOG` will log verbose output from Terraform commands to help debug issues